collections/esdc-8d-0.083deg-1x2160x4320-2.1.1.yaml

```diff
@@ -0,0 +1,205 @@
+          Description: Net Radiation
+        - Name: open_water_evaporation
+          Description: Open-water Evaporation
+        - Name: ozone
+          Description: Mean Total Ozone Column in dobson units
+        - Name: par
+          Description: Photosynthetically Active Radiation
+        - Name: pardiff
+          Description: Diffuse Photosynthetically Active Radiation
+        - Name: potential_evaporation
+          Description: Potential Evaporation
+        - Name: precipitation
+          Description: Precipitation
+        - Name: precipitation_era5
+          Description: ERA 5 Precipitation
+        - Name: psurf
+          Description: Surface Air Pressure
+        - Name: radiation_era5
+          Description: ERA 5 Surface Shortwave Downwelling Radiation
+        - Name: Rg
+          Description: Downwelling shortwave radiation
+        - Name: root_moisture
+          Description: Root-Zone Soil Moisture
+        - Name: sea_ice_fraction
+          Description: sea ice area fraction
+        - Name: sensible_heat
+          Description: Sensible Heat
+        - Name: sm
+          Description: Volumetric Soil Moisture
+        - Name: snow_sublimation
+          Description: Snow Sublimation
+        - Name: snow_water_equivalent
+          Description: Daily Snow Water Equivalent
+        - Name: srex_mask
+          Description: Mask for SREX Regions
+        - Name: stemp
+          Description: Surface Temperature
+        - Name: surface_moisture
+          Description: Surface Soil Moisture
+        - Name: terrestrial_ecosystem_respiration
+          Description: Terrestrial Ecosystem Respiration
+        - Name: totcol_assim
+          Description: Total Ozone Column (Assimilated TM5 data)
+        - Name: totcol_free
+          Description: Total Ozone Column (Assimilated TM5 data)
+        - Name: totcol_msr
+          Description: Total Ozone Column (MSR data)
+        - Name: transpiration
+          Description: Transpiration
+        - Name: water_mask
+          Description: Terrestrial or Water Pixel Classification
+        - Name: water_vapour
+          Description: Total Column Water Vapour
+        - Name: white_sky_albedo
+          Description: White Sky Albedo for Visible Wavebands
+        - Name: white_sky_albedo_avhrr
+          Description: Bi-Hemisphere Reflectance Albedo - VIS band
+        - Name: xch4
+          Description: Column Average Dry-air Mole Fraction Methane
+        - Name: xco2
+          Description: Column Average Dry-air Mole Fraction Carbon Dioxide
+
+Contact: info@earthsystemdatalab.net
+Provider: "[Earth System Data Lab](https://www.earthsystemdatalab.net/)"
+ManagedBy: "[Brockmann Consult](https://www.brockmann-consult.de/)"
+Tags:
+  - Earth System Data Cube
+  - xcube
+  - raster
+  - systematic
+  - open data
+License: |
+  Please refer to individual variables.
+Resources:
+-  Group: xcube Resources
+   StoreID: s3
+   Root: esdl-esdc-v2.1.1
```

Comment:
The handler does not recognise `StoreID` and `Root`. Please refer to the metadata table in the [contributions file](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md):

| Field | Type | Description & Style |
| --- | --- | --- |
|**Resources > DatasetName**|String|Name of the xcube dataset.|
|**Resources > StoreInstanceID**|String|ID of the xcube store.|
|**Resources > StoreTitle**|String|Name of xcube store of the collection.|
|**Resources > DataID**|String|Name identifier of the collection in the xcube store.|
|**Resources > Notes**|MD| More info regarding the collection.|

Current view:
![image](https://user-images.githubusercontent.com/60575896/205029210-215cda56-c6dc-45b5-8c34-cf946db8d350.png)